### DISCUSSION TITLE
Is there a way to "anchor" a background image via Bootstrap?

### DISCUSSION THREAD

**Discussion by paul-garrett** [twbs/bootstrap#35811] [Q&A]
I'm hoping that I can just anchor my background image. I'm not a fan of the seam when an image repeats itself, or there just being a white background once enough text has covered the maximum size of the image. Is there a way in Bootstrap to do this? I'm creating a message board and want the text to scroll in front of the anchored image. I haven't had any luck with Google or others I know that have helped in the past. Thanks for any help anyone has!

**Comment by ascensus-mdb** [comment]
Have a look a this https://mdbootstrap.com/docs/standard/content-styles/background-image/

**Reply by paul-garrett** [reply]
That's anchoring it!

Unfortunately it's still giving me a white background just past the text. Is there a way to add a "fit to screen" type of command to it?

**Comment by mariacleh7** [comment]
Yes I have used anchor a background image via bootstrap.  You have to add display:block or display:inline-block as a tags are set to display:inline by default and inline elements can't have a width , height or margin . To get the background image working you have to add url . Please note that this url is relative to your css file. Moreover visit [slots free chips](https://telecombit.com/pop-slots-free-chips/) for more guidance.

**Reply by paul-garrett** [reply]
So would I add that to my style tag? That link you provided sends me to a poker game.
